dts/arm/st/stm32f1.dtsi

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/stm32f1-pinctrl.dtsi>
+#include <arm/armv7-m.dtsi>
+#include <st/mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m3";
+			reg = <0>;
+		};
+	};
+
+	flash0: flash@8000000 {
+		reg = <0x08000000 DT_FLASH_SIZE>;
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 DT_SRAM_SIZE>;
+	};
+
+	soc {
+		pinctrl: pin-controller {
+			compatible = "st,stm32-pinmux";
```

Comment:
I'm not sure why we should use 0 for size cells.
When we'll add gpio nodes, we'll have reg = <address size>;
Can you clarify?